backend/clubs/views.py

```diff
@@ -2487,7 +2537,11 @@ def tickets(self, request, *args, **kwargs):
             available.append(
                 {
                     "type": type,
-                    "count": (tickets.filter(type=type, owner__isnull=True).count()),
+                    "count": (
+                        tickets.filter(
+                            type=type, owner__isnull=True, holder__isnull=True
+                        ).count()
+                    ),
                 }
             )
```

Comment:
Does something like this work instead? 
```
totals = tickets.values('type').annotate(count=Count('type')).order_by('type')
available = tickets.filter(owner__isnull=True, holder__isnull=True).values('type').annotate(count=Count('type')).order_by('type')
return Response({"totals": list(totals), "available": list(available)})
```

Reply:
Yep, seems good to me. I'll add it in, thanks - that's another TODO gone